packages/sdk/README.md

```diff
@@ -711,17 +712,6 @@ client.customers.listActivities(customerId, params)
 client.customers.importCSV(filePath)
 ```
 
-#### [Update Customers in bulk](https://docs.voucherify.io/reference/update-customers-in-bulk)
```

Comment:
Why do we remove those examples?

Reply:
ahh these are problems after merging with main